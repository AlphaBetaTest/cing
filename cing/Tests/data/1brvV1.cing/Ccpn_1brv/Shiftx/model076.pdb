REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       5.295  -0.960   4.534  1.00  0.00              
ATOM      2  CA  VAL A 171       4.607  -0.168   3.489  1.00  0.00              
ATOM      3  HA  VAL A 171       4.358   0.836   3.881  1.00  0.00              
ATOM      4  CB  VAL A 171       5.648   0.062   2.339  1.00  0.00              
ATOM      5  HB  VAL A 171       5.062   0.543   1.556  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.784   1.049   2.693  1.00  0.00              
ATOM      7 HG11 VAL A 171       6.393   2.007   3.083  1.00  0.00              
ATOM      8 HG12 VAL A 171       7.467   0.638   3.461  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.403   1.293   1.809  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.275  -1.185   1.667  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.504  -1.916   1.363  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.835  -0.914   0.751  1.00  0.00              
ATOM     13 HG23 VAL A 171       6.978  -1.715   2.336  1.00  0.00              
ATOM     14  C   VAL A 171       3.214  -0.784   3.064  1.00  0.00              
ATOM     15  O   VAL A 171       3.110  -1.319   1.954  1.00  0.00              
ATOM     16  N   PRO A 172       2.096  -0.700   3.857  1.00  0.00              
ATOM     17  CD  PRO A 172       2.145  -0.375   5.301  1.00  0.00              
ATOM     18  CA  PRO A 172       0.774  -1.245   3.439  1.00  0.00              
ATOM     19  HA  PRO A 172       0.898  -2.073   2.710  1.00  0.00              
ATOM     20  CB  PRO A 172       0.295  -1.877   4.759  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.806  -1.975   4.824  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.701  -2.906   4.842  1.00  0.00              
ATOM     23  CG  PRO A 172       0.878  -1.012   5.880  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.164  -0.228   6.178  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.088  -1.601   6.793  1.00  0.00              
ATOM     26  HD2 PRO A 172       2.163   0.720   5.469  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.030  -0.805   5.806  1.00  0.00              
ATOM     28  C   PRO A 172      -0.206  -0.175   2.836  1.00  0.00              
ATOM     29  O   PRO A 172      -0.712   0.710   3.537  1.00  0.00              
ATOM     30  N   CYS A 173      -0.525  -0.309   1.536  1.00  0.00              
ATOM     31  H   CYS A 173       0.030  -0.986   0.998  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.491   0.561   0.796  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.115   1.585   0.740  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.444   0.073  -0.659  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.949  -0.895  -0.750  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.454   0.003  -1.041  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.458   1.067  -1.779  1.00  0.00              
ATOM     38  C   CYS A 173      -2.933   0.700   1.403  1.00  0.00              
ATOM     39  O   CYS A 173      -3.523   1.782   1.322  1.00  0.00              
ATOM     40  N   SER A 174      -3.472  -0.359   2.040  1.00  0.00              
ATOM     41  H   SER A 174      -2.853  -1.177   2.058  1.00  0.00              
ATOM     42  CA  SER A 174      -4.720  -0.276   2.869  1.00  0.00              
ATOM     43  HA  SER A 174      -5.558  -0.021   2.222  1.00  0.00              
ATOM     44  CB  SER A 174      -4.994  -1.681   3.455  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.974  -1.683   3.973  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.098  -2.421   2.639  1.00  0.00              
ATOM     47  OG  SER A 174      -3.973  -2.106   4.364  1.00  0.00              
ATOM     48  HG  SER A 174      -4.060  -1.567   5.159  1.00  0.00              
ATOM     49  C   SER A 174      -4.783   0.824   3.980  1.00  0.00              
ATOM     50  O   SER A 174      -5.857   1.319   4.321  1.00  0.00              
ATOM     51  N   THR A 175      -3.619   1.184   4.520  1.00  0.00              
ATOM     52  H   THR A 175      -2.813   0.717   4.087  1.00  0.00              
ATOM     53  CA  THR A 175      -3.455   2.255   5.546  1.00  0.00              
ATOM     54  HA  THR A 175      -4.417   2.368   6.085  1.00  0.00              
ATOM     55  CB  THR A 175      -2.412   1.778   6.601  1.00  0.00              
ATOM     56  HB  THR A 175      -2.472   2.470   7.466  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.084   1.839   6.086  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.088   1.413   5.218  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.652   0.342   7.118  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.707   0.204   7.427  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.468  -0.405   6.320  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.998   0.087   7.964  1.00  0.00              
ATOM     63  C   THR A 175      -3.144   3.676   4.949  1.00  0.00              
ATOM     64  O   THR A 175      -3.416   4.685   5.601  1.00  0.00              
ATOM     65  N   CYS A 176      -2.568   3.769   3.728  1.00  0.00              
ATOM     66  H   CYS A 176      -2.614   2.911   3.173  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.412   5.039   2.986  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.958   5.805   3.646  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.413   4.675   1.872  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.776   3.951   1.148  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.212   5.525   1.206  1.00  0.00              
ATOM     72  SG  CYS A 176       0.142   4.112   2.593  1.00  0.00              
ATOM     73  C   CYS A 176      -3.788   5.608   2.493  1.00  0.00              
ATOM     74  O   CYS A 176      -4.035   6.808   2.613  1.00  0.00              
ATOM     75  N   GLU A 177      -4.662   4.737   1.941  1.00  0.00              
ATOM     76  H   GLU A 177      -4.232   3.852   1.661  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.091   5.016   1.625  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.345   4.235   0.891  1.00  0.00              
ATOM     79  CB  GLU A 177      -6.939   4.696   2.888  1.00  0.00              
ATOM     80  HB2 GLU A 177      -7.996   4.688   2.569  1.00  0.00              
ATOM     81  HB3 GLU A 177      -6.709   3.659   3.194  1.00  0.00              
ATOM     82  CG  GLU A 177      -6.808   5.624   4.128  1.00  0.00              
ATOM     83  HG2 GLU A 177      -5.762   5.653   4.482  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.056   6.668   3.861  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.668   5.193   5.310  1.00  0.00              
ATOM     86  OE1 GLU A 177      -8.918   5.726   5.263  1.00  0.00              
ATOM     87  OE2 GLU A 177      -7.284   4.449   6.206  1.00  0.00              
ATOM     88  C   GLU A 177      -6.482   6.330   0.890  1.00  0.00              
ATOM     89  O   GLU A 177      -7.449   7.026   1.210  1.00  0.00              
ATOM     90  N   GLY A 178      -5.723   6.590  -0.170  1.00  0.00              
ATOM     91  H   GLY A 178      -4.813   6.129  -0.082  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.848   7.831  -0.953  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.839   7.518  -2.007  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.821   8.305  -0.787  1.00  0.00              
ATOM     95  C   GLY A 178      -4.743   8.896  -0.775  1.00  0.00              
ATOM     96  O   GLY A 178      -4.767   9.902  -1.485  1.00  0.00              
ATOM     97  N   ASN A 179      -3.761   8.688   0.126  1.00  0.00              
ATOM     98  H   ASN A 179      -3.708   7.714   0.456  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.597   9.586   0.259  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.858  10.620  -0.049  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.978   9.473   1.666  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.735   9.285   2.452  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.321   8.578   1.622  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.089  10.651   2.030  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.076  10.722   1.659  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.617  11.619   2.723  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.597  11.502   2.994  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.972  12.383   2.934  1.00  0.00              
ATOM    109  C   ASN A 179      -1.580   9.081  -0.750  1.00  0.00              
ATOM    110  O   ASN A 179      -0.930   8.044  -0.575  1.00  0.00              
ATOM    111  N   LEU A 180      -1.431   9.842  -1.808  1.00  0.00              
ATOM    112  H   LEU A 180      -2.031  10.668  -1.865  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.631   9.380  -2.934  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.052   8.371  -3.038  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.039  10.327  -4.067  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.170  11.358  -3.660  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.197  10.343  -4.726  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.246   9.932  -4.928  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.037   9.759  -4.201  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.663  11.082  -5.857  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.888  12.007  -5.295  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.874  11.330  -6.593  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.576  10.834  -6.431  1.00  0.00              
ATOM    124  CD2 LEU A 180      -2.023   8.656  -5.761  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.788   7.778  -5.132  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.922   8.388  -6.347  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.190   8.770  -6.480  1.00  0.00              
ATOM    128  C   LEU A 180       0.905   9.161  -2.739  1.00  0.00              
ATOM    129  O   LEU A 180       1.519   8.373  -3.452  1.00  0.00              
ATOM    130  N   ALA A 181       1.476   9.840  -1.749  1.00  0.00              
ATOM    131  H   ALA A 181       0.745  10.361  -1.267  1.00  0.00              
ATOM    132  CA  ALA A 181       2.844   9.576  -1.252  1.00  0.00              
ATOM    133  HA  ALA A 181       3.501   9.475  -2.127  1.00  0.00              
ATOM    134  CB  ALA A 181       3.297  10.792  -0.453  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.277  11.703  -1.073  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.654  10.942   0.436  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.332  10.636  -0.097  1.00  0.00              
ATOM    138  C   ALA A 181       2.949   8.264  -0.404  1.00  0.00              
ATOM    139  O   ALA A 181       3.820   7.443  -0.677  1.00  0.00              
ATOM    140  N   CYS A 182       2.060   8.054   0.594  1.00  0.00              
ATOM    141  H   CYS A 182       1.233   8.657   0.509  1.00  0.00              
ATOM    142  CA  CYS A 182       1.936   6.755   1.326  1.00  0.00              
ATOM    143  HA  CYS A 182       2.805   6.518   1.906  1.00  0.00              
ATOM    144  CB  CYS A 182       0.885   7.011   2.399  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.028   7.980   2.885  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.100   7.070   1.942  1.00  0.00              
ATOM    147  SG  CYS A 182       0.836   5.677   3.603  1.00  0.00              
ATOM    148  C   CYS A 182       1.707   5.523   0.394  1.00  0.00              
ATOM    149  O   CYS A 182       2.359   4.481   0.493  1.00  0.00              
ATOM    150  N   LEU A 183       0.786   5.717  -0.544  1.00  0.00              
ATOM    151  H   LEU A 183       0.262   6.598  -0.448  1.00  0.00              
ATOM    152  CA  LEU A 183       0.518   4.817  -1.669  1.00  0.00              
ATOM    153  HA  LEU A 183       0.262   3.841  -1.269  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.715   5.456  -2.313  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.493   6.502  -2.558  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.846   5.021  -3.259  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.085   5.310  -1.652  1.00  0.00              
ATOM    158  HG  LEU A 183      -1.973   5.842  -0.692  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.120   6.001  -2.548  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.078   5.625  -3.589  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.148   5.829  -2.197  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.954   7.092  -2.593  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.497   3.833  -1.481  1.00  0.00              
ATOM    164 HD21 LEU A 183      -1.772   3.259  -0.880  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.470   3.722  -0.972  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.561   3.303  -2.451  1.00  0.00              
ATOM    167  C   LEU A 183       1.687   4.578  -2.679  1.00  0.00              
ATOM    168  O   LEU A 183       1.825   3.474  -3.203  1.00  0.00              
ATOM    169  N   SER A 184       2.507   5.606  -2.937  1.00  0.00              
ATOM    170  H   SER A 184       2.268   6.408  -2.336  1.00  0.00              
ATOM    171  CA  SER A 184       3.783   5.450  -3.702  1.00  0.00              
ATOM    172  HA  SER A 184       3.565   4.882  -4.629  1.00  0.00              
ATOM    173  CB  SER A 184       4.295   6.839  -4.137  1.00  0.00              
ATOM    174  HB2 SER A 184       3.506   7.392  -4.684  1.00  0.00              
ATOM    175  HB3 SER A 184       4.544   7.459  -3.254  1.00  0.00              
ATOM    176  OG  SER A 184       5.442   6.716  -4.983  1.00  0.00              
ATOM    177  HG  SER A 184       5.961   5.972  -4.647  1.00  0.00              
ATOM    178  C   SER A 184       4.903   4.631  -2.956  1.00  0.00              
ATOM    179  O   SER A 184       5.820   4.134  -3.614  1.00  0.00              
ATOM    180  N   LEU A 185       4.866   4.533  -1.612  1.00  0.00              
ATOM    181  H   LEU A 185       4.028   4.965  -1.188  1.00  0.00              
ATOM    182  CA  LEU A 185       5.752   3.612  -0.834  1.00  0.00              
ATOM    183  HA  LEU A 185       6.745   3.541  -1.294  1.00  0.00              
ATOM    184  CB  LEU A 185       5.897   4.101   0.638  1.00  0.00              
ATOM    185  HB2 LEU A 185       4.938   3.914   1.159  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.627   3.449   1.151  1.00  0.00              
ATOM    187  CG  LEU A 185       6.298   5.575   0.883  1.00  0.00              
ATOM    188  HG  LEU A 185       5.548   6.198   0.354  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.235   5.915   2.380  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.225   5.738   2.795  1.00  0.00              
ATOM    191 HD12 LEU A 185       6.946   5.313   2.976  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.470   6.979   2.564  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.698   5.905   0.340  1.00  0.00              
ATOM    194 HD21 LEU A 185       8.487   5.286   0.809  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.759   5.740  -0.752  1.00  0.00              
ATOM    196 HD23 LEU A 185       7.964   6.964   0.510  1.00  0.00              
ATOM    197  C   LEU A 185       5.281   2.133  -0.910  1.00  0.00              
ATOM    198  O   LEU A 185       6.099   1.223  -1.077  1.00  0.00              
ATOM    199  N   CYS A 186       3.970   1.899  -0.773  1.00  0.00              
ATOM    200  H   CYS A 186       3.414   2.671  -0.414  1.00  0.00              
ATOM    201  CA  CYS A 186       3.381   0.582  -0.903  1.00  0.00              
ATOM    202  HA  CYS A 186       4.084  -0.083  -0.404  1.00  0.00              
ATOM    203  CB  CYS A 186       2.076   0.681  -0.095  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.566  -0.274  -0.132  1.00  0.00              
ATOM    205  HB3 CYS A 186       2.125   0.917   0.963  1.00  0.00              
ATOM    206  SG  CYS A 186       0.990   1.892  -0.725  1.00  0.00              
ATOM    207  C   CYS A 186       3.114   0.037  -2.348  1.00  0.00              
ATOM    208  O   CYS A 186       3.280   0.669  -3.392  1.00  0.00              
ATOM    209  N   HIS A 187       2.617  -1.197  -2.327  1.00  0.00              
ATOM    210  H   HIS A 187       2.773  -1.640  -1.417  1.00  0.00              
ATOM    211  CA  HIS A 187       2.189  -1.967  -3.515  1.00  0.00              
ATOM    212  HA  HIS A 187       2.688  -1.590  -4.423  1.00  0.00              
ATOM    213  CB  HIS A 187       2.622  -3.448  -3.339  1.00  0.00              
ATOM    214  HB2 HIS A 187       1.930  -3.933  -2.660  1.00  0.00              
ATOM    215  HB3 HIS A 187       2.469  -3.961  -4.291  1.00  0.00              
ATOM    216  CG  HIS A 187       4.029  -3.749  -2.832  1.00  0.00              
ATOM    217  ND1 HIS A 187       4.327  -3.793  -1.480  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.159  -3.455  -3.583  1.00  0.00              
ATOM    219  CE1 HIS A 187       5.650  -3.457  -1.542  1.00  0.00              
ATOM    220  NE2 HIS A 187       6.257  -3.277  -2.758  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.052  -3.136  -4.604  1.00  0.00              
ATOM    222  HE1 HIS A 187       6.120  -3.117  -0.633  1.00  0.00              
ATOM    223  C   HIS A 187       0.637  -1.823  -3.647  1.00  0.00              
ATOM    224  O   HIS A 187      -0.105  -2.095  -2.694  1.00  0.00              
ATOM    225  N   ILE A 188       0.113  -1.493  -4.836  1.00  0.00              
ATOM    226  H   ILE A 188       0.806  -1.226  -5.540  1.00  0.00              
ATOM    227  CA  ILE A 188      -1.340  -1.163  -4.989  1.00  0.00              
ATOM    228  HA  ILE A 188      -1.726  -0.753  -4.038  1.00  0.00              
ATOM    229  CB  ILE A 188      -1.555   0.033  -5.987  1.00  0.00              
ATOM    230  HB  ILE A 188      -1.103  -0.268  -6.947  1.00  0.00              
ATOM    231  CG2 ILE A 188      -3.060   0.312  -6.264  1.00  0.00              
ATOM    232 HG21 ILE A 188      -3.579  -0.570  -6.684  1.00  0.00              
ATOM    233 HG22 ILE A 188      -3.609   0.603  -5.349  1.00  0.00              
ATOM    234 HG23 ILE A 188      -3.206   1.120  -7.006  1.00  0.00              
ATOM    235  CG1 ILE A 188      -0.940   1.403  -5.547  1.00  0.00              
ATOM    236 HG12 ILE A 188      -1.397   2.229  -6.125  1.00  0.00              
ATOM    237 HG13 ILE A 188      -1.210   1.623  -4.496  1.00  0.00              
ATOM    238  CD1 ILE A 188       0.580   1.565  -5.728  1.00  0.00              
ATOM    239 HD11 ILE A 188       0.918   1.283  -6.742  1.00  0.00              
ATOM    240 HD12 ILE A 188       0.891   2.615  -5.568  1.00  0.00              
ATOM    241 HD13 ILE A 188       1.163   0.974  -4.999  1.00  0.00              
ATOM    242  C   ILE A 188      -2.146  -2.470  -5.317  1.00  0.00              
ATOM    243  O   ILE A 188      -2.253  -2.925  -6.459  1.00  0.00              
ATOM    244  N   GLU A 189      -2.727  -3.047  -4.253  1.00  0.00              
ATOM    245  H   GLU A 189      -2.353  -2.706  -3.364  1.00  0.00              
ATOM    246  CA  GLU A 189      -3.548  -4.283  -4.313  1.00  0.00              
ATOM    247  HA  GLU A 189      -2.998  -5.014  -4.936  1.00  0.00              
ATOM    248  CB  GLU A 189      -3.601  -4.872  -2.872  1.00  0.00              
ATOM    249  HB2 GLU A 189      -4.066  -5.877  -2.923  1.00  0.00              
ATOM    250  HB3 GLU A 189      -2.564  -5.075  -2.537  1.00  0.00              
ATOM    251  CG  GLU A 189      -4.340  -4.039  -1.780  1.00  0.00              
ATOM    252  HG2 GLU A 189      -3.969  -2.998  -1.768  1.00  0.00              
ATOM    253  HG3 GLU A 189      -5.413  -3.955  -2.029  1.00  0.00              
ATOM    254  CD  GLU A 189      -4.238  -4.612  -0.369  1.00  0.00              
ATOM    255  OE1 GLU A 189      -4.835  -5.614   0.008  1.00  0.00              
ATOM    256  OE2 GLU A 189      -3.416  -3.883   0.435  1.00  0.00              
ATOM    257  C   GLU A 189      -4.955  -4.098  -4.962  1.00  0.00              
TER     258      GLU A 189 
END   
