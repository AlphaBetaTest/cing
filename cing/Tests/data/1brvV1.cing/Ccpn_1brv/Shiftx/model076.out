# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0579
  171   V  CA      60.7411
  171   V  CB      32.4408
  171   V  C      172.9846
  171   V  H        8.0393
  171   V  HA       4.0050
  171   V  HB       2.0869
  171   V  HG11     0.9326
  171   V  HG12     0.9616
  171   V  HG13     0.8360
  171   V  HG21     0.9700
  171   V  HG22     1.0318
  171   V  HG23     0.9614
  172   P  CA      60.6416
  172   P  CB      32.7731
  172   P  C      176.6624
  172   P  HA       4.5436
  172   P  HB2      2.2914
  172   P  HB3      2.2000
  172   P  HD2      3.7106
  172   P  HD3      3.7729
  172   P  HG2      2.0017
  172   P  HG3      2.0513
  173   C  N      122.5314
  173   C  CA      59.9939
  173   C  CB      40.7674
  173   C  C      177.1891
  173   C  H        8.8001
  173   C  HA       4.2271
  173   C  HB2      3.0752
  173   C  HB3      3.1895
  174   S  N      114.8057
  174   S  CA      61.5269
  174   S  CB      62.2562
  174   S  C      175.9615
  174   S  H        8.2133
  174   S  HA       3.9774
  174   S  HB2      4.0164
  174   S  HB3      4.0262
  175   T  N      111.5574
  175   T  CA      63.5272
  175   T  CB      69.4063
  175   T  C      176.2074
  175   T  H        7.6525
  175   T  HA       4.2560
  175   T  HB       4.2399
  175   T  HG21     1.2111
  175   T  HG22     1.2118
  175   T  HG23     1.2114
  176   C  N      119.6354
  176   C  CA      58.5369
  176   C  CB      40.7355
  176   C  C      174.0253
  176   C  H        8.1817
  176   C  HA       4.3636
  176   C  HB2      3.1292
  176   C  HB3      3.2145
  177   E  N      115.9405
  177   E  CA      57.5222
  177   E  CB      28.4505
  177   E  C      175.3933
  177   E  H        9.1431
  177   E  HA       3.5339
  177   E  HB2      2.2102
  177   E  HB3      2.1979
  177   E  HG2      2.3713
  177   E  HG3      2.2306
  178   G  N      104.3589
  178   G  CA      46.2489
  178   G  C      173.7971
  178   G  H        8.3691
  178   G  HA    -666.0000
  178   G  HA2      3.9581
  178   G  HA3      3.9576
  179   N  N      119.1111
  179   N  CA      52.5182
  179   N  CB      39.3397
  179   N  C      175.2914
  179   N  H        8.1826
  179   N  HA       4.5872
  179   N  HB2      2.8824
  179   N  HB3      2.7261
  179   N  HD21     7.1220
  179   N  HD22     6.6632
  180   L  N      129.0754
  180   L  CA      57.2295
  180   L  CB      41.6317
  180   L  C      179.4312
  180   L  H        8.4634
  180   L  HA       3.9877
  180   L  HB2      1.7227
  180   L  HB3      1.6993
  180   L  HD11     0.9177
  180   L  HD12     0.9175
  180   L  HD13     0.9178
  180   L  HD21     0.2352
  180   L  HD22     1.7187
  180   L  HD23     0.8662
  180   L  HG       1.7034
  181   A  N      120.7978
  181   A  CA      55.2059
  181   A  CB      18.4490
  181   A  C      179.6986
  181   A  H        7.8959
  181   A  HA       3.9131
  181   A  HB1      1.3850
  181   A  HB2      1.3460
  181   A  HB3      1.3287
  182   C  N      116.3733
  182   C  CA      60.1028
  182   C  CB      43.4395
  182   C  C      175.8932
  182   C  H        7.9693
  182   C  HA       4.1250
  182   C  HB2      3.0148
  182   C  HB3      3.2257
  183   L  N      119.3615
  183   L  CA      57.8399
  183   L  CB      41.2462
  183   L  C      179.2523
  183   L  H        7.9990
  183   L  HA       3.8721
  183   L  HB2      1.8610
  183   L  HB3      1.7083
  183   L  HD11     0.9149
  183   L  HD12     0.9912
  183   L  HD13     0.9152
  183   L  HD21     0.3193
  183   L  HD22     1.6368
  183   L  HD23     0.8629
  183   L  HG       1.6652
  184   S  N      112.0659
  184   S  CA      60.7444
  184   S  CB      62.3080
  184   S  C      175.9393
  184   S  H        7.9236
  184   S  HA       4.2174
  184   S  HB2      3.9916
  184   S  HB3      4.0217
  185   L  N      121.8412
  185   L  CA      57.0509
  185   L  CB      41.7379
  185   L  C      174.9338
  185   L  H        7.4811
  185   L  HA       4.1595
  185   L  HB2      1.7050
  185   L  HB3      1.6855
  185   L  HD11     0.9114
  185   L  HD12     0.9133
  185   L  HD13     0.9136
  185   L  HD21     0.2946
  185   L  HD22     1.5263
  185   L  HD23     0.8630
  185   L  HG       1.8112
  186   C  N      111.7764
  186   C  CA      54.7838
  186   C  CB      43.0988
  186   C  C      171.9086
  186   C  H        7.5373
  186   C  HA       4.5606
  186   C  HB2      3.0617
  186   C  HB3      3.0707
  187   H  N      116.5737
  187   H  CA      54.4290
  187   H  CB      29.3884
  187   H  C      175.7412
  187   H  H        8.1181
  187   H  HA       4.6851
  187   H  HB2      3.1218
  187   H  HB3      3.2842
  187   H  HD2      5.8839
  187   H  HE1      6.8807
  188   I  N      128.0662
  188   I  CA      60.6878
  188   I  CB      36.1687
  188   I  C      176.3588
  188   I  H        8.2352
  188   I  HA       3.9432
  188   I  HB       1.8722
  188   I  HD11     0.8525
  188   I  HD12     0.8444
  188   I  HD13     0.8364
  188   I  HG12     1.2681
  188   I  HG13     0.3667
  188   I  HG21     0.9129
  188   I  HG22     0.9102
  188   I  HG23     0.9102
 *189   E  N      120.2923
 *189   E  CA      55.9416
 *189   E  CB      30.2534
 *189   E  C      173.8599
 *189   E  H        8.4738
 *189   E  HA       4.1626
 *189   E  HB2      1.7406
 *189   E  HB3      1.7773
 *189   E  HG2      2.1351
 *189   E  HG3      2.3797
